website/blog/2021-08-06-version-065.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Announcing React Native 0.65
+author:
+authorTitle:
+authorURL:
+authorImageURL:
+authorTwitter:
+tags: [announcement, release]
+---
+
+Today we’re releasing React Native version 0.65 with a new version of Hermes, improvements to accessibility, package upgrades, and more.
+
+## What's new in Hermes 0.8?
+
+[Hermes](https://hermesengine.dev), Facebook’s open source JavaScript VM optimized for React Native, has been upgraded to version 0.8.1. Some of the stand-out features in this release are:
+
+- A new concurrent garbage collector titled “Hades” which delivers up to 30 times shorter pause times on 64 bit devices. At Facebook, we saw this improve some CPU-intensive workloads by 20%-50%. You can [learn more about Hades here](https://hermesengine.dev/docs/hades/).
+- [ECMAScript Internationalization API (ECMA-402, or `Intl`)](https://hermesengine.dev/docs/intl) is now built into Hermes on Android and enabled by default, with only 57-62K per API size overhead (compared to [JSC's 6MiB](https://github.com/react-native-community/jsc-android-buildscripts)). With this change, Hermes users no longer require locale polyfills. A big thank you to [@mganandraj](https://github.com/mganandraj) and other fellow partners at Microsoft for driving the implementation to make this happen!
```

Comment:
Perhaps APK instead of API?

Reply:
cc @Huxpro would that be accurate?

Reply:
I think @dulmandakh is correct here - looping @mganandraj too for confirmation